src/ia32/lithium-codegen-ia32.cc

```diff
@@ -6406,33 +6406,6 @@ void LCodeGen::DoUnarySIMDOperation(LUnarySIMDOperation* instr) {
       __ movmskps(result, input_reg);
       return;
     }
-    case kFloat32x4GetW:
-      select++;
-    case kFloat32x4GetZ:
-      select++;
-    case kFloat32x4GetY:
-      select++;
-    case kFloat32x4GetX: {
-      DCHECK(instr->hydrogen()->value()->representation().IsFloat32x4());
-      XMMRegister input_reg = ToFloat32x4Register(instr->value());
-      XMMRegister result = ToDoubleRegister(instr->result());
-      XMMRegister xmm_scratch = result.is(input_reg) ? xmm0 : result;
-
-      if (select == 0x0) {
-        __ xorps(xmm_scratch, xmm_scratch);
-        __ cvtss2sd(xmm_scratch, input_reg);
-        if (!xmm_scratch.is(result)) {
-          __ movaps(result, xmm_scratch);
-        }
-      } else {
-        __ pshufd(xmm_scratch, input_reg, select);
-        if (!xmm_scratch.is(result)) {
-           __ xorps(result, result);
-        }
-        __ cvtss2sd(result, xmm_scratch);
-      }
-      return;
-    }
```

Comment:
yes， I will implement this in the crankshaft patch. And I will send the PR which will include implementation of runtime and crankshaft